evergreen/mongodl.py

```diff
@@ -0,0 +1,938 @@
+#!/usr/bin/env python3
```

Comment:
this file should be called from `EVERGREEN_TOOLS` https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/mongodl.py instead of adding this under source control

Reply:
btw it looks like you actually use it from this path based on `${DRIVERS_TOOLS}/.evergreen/mongodl.py` :)

Reply:
Good point. Removed.